ocaml/xapi-cli-server/cli_operations.ml

```diff
@@ -7845,7 +7845,14 @@ module VTPM = struct
   let create printer rpc session_id params =
     let vm_uuid = List.assoc "vm-uuid" params in
     let vM = Client.VM.get_by_uuid ~rpc ~session_id ~uuid:vm_uuid in
-    let ref = Client.VTPM.create ~rpc ~session_id ~vM in
+    let unique =
+      match List.assoc_opt "unique" params with
+      | Some value ->
+          Stdlib.bool_of_string value
```

Comment:
This can fail. Is this string user supplied? Likewise instances below. It might be safer to use a function that catches exceptions.

Reply:
I've replaced it with the module's own 'bool_of_string' to have a better error